drfdjango/settings.py

```diff
@@ -0,0 +1,130 @@
+"""
+Django settings for drfdjango project.
+
+Generated by 'django-admin startproject' using Django 4.2.4.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-82m1wm-ty*)2u^j15d)qadwzmkwv4n*&+=8wg0%5*fy@r85@y9'
```

Comment:
this should not be committed, you can use ENV variables to hide sensitive information